spec/requests/sample_api_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rails_helper'
```

Comment:
why is this called sample api spec? Is it a request type of test kind of thing?

Reply:
Yeah it's a request spec, I suppose `sample_request_spec` is a better name :)

Reply:
I dont think you need it: `sample_spec.rb` is fine because it is already inside the requests folder

Reply:
Unless there is a conflict...

Reply:
`sample_spec.rb` is the name of the Model's spec test file, I don't want to cause any confusion.